test/topology_custom/test_multidc.py

```diff
@@ -0,0 +1,209 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""
+Test functionality on the cluster with different values of the --smp parameter on the nodes.
```

Comment:
I don't understand the relevance of this comment. Maybe it's a copypasto?

Reply:
Correct, this is a copypaste from the https://github.com/kostja/scylla/tree/multidc2. There was one test where @kostja  made the initial commit to support multi-DC. I'll delete this comment to remove confusion.